admin_manual/configuration_files/encryption_configuration.rst

```diff
@@ -147,13 +147,14 @@ Encryption settings can be configured in the mount options for an external stora
 How To Enable Users File Recovery Keys
 --------------------------------------
 
-If you lose your ownCloud password, then you lose access to your encrypted files. 
-If one of your users loses their ownCloud password, their files are unrecoverable. 
-You cannot reset their password in the normal way, however. 
+Once a user has encrypted their files, if they lose their ownCloud password, then they lose access to their encrypted files, as their files will be unrecoverable. 
+It is not possible, when user files are encrypted, to reset a user’s password using the standard reset process. 
 
-You'll see a yellow banner warning "Please provide an admin recovery password; otherwise, all user data will be lost."
-To avoid all this, create a Recovery Key. 
-To do so, go to the Encryption section of your Admin page and set a recovery key password.
+If so, you'll see a yellow banner warning: 
+
+  Please provide an admin recovery password; otherwise, all user data will be lost.
+
+To avoid all this, create a Recovery Key. To do so, go to the Encryption section of your Admin page and set a recovery key password.
```

Comment:
You also need to ask your users to opt-in for the recovery key. They need to go to the Personal page and enable recovery key to signal they are ok that the admin might have a way to decrypt their data for recovery reasons.